### PR TITLE
[PIN-1162] Fixing dependency resolution

### PR DESCRIPTION
**Pull request by TonioGela**

### CODE REVIEW

File: .sbtopts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-Djavax.net.ssl.trustStore=/Users/toniogela/.config/PDNDTrustStore
```
Comment (ubaldop) [inline .sbtopts]: is this necessary?

**Review by ubaldop — APPROVED** [review]

File: .sbtopts (hunk 1 above)
Comment (beetlecrunch) [inline .sbtopts]: ⚠️ remove from commit and update the .gitignore file

Comment (TonioGela) [reply .sbtopts]: my bad, I'll remove it

**Review by galales — APPROVED** [review]

File: .sbtopts (hunk 1 above)
Comment (TonioGela) [reply .sbtopts]: yep, it was a local test, I'll squash the commit to remove it from history too

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**